int64 robotID
int64 taskID
int64 action
geometry_msgs/Point start
geometry_msgs/Point goal
marmot/TaskAck[] allFrom
marmot/TaskAck[] allTo